PhysicsTools/PatAlgos/python/producersLayer1/jetProducer_cff.py

```diff
@@ -18,3 +18,19 @@
     patJets
     )
 makePatJets = cms.Sequence(makePatJetsTask)
+
+from RecoBTag.ImpactParameter.pfImpactParameterTagInfos_cfi import * #pfImpactParameterTagInfos
+from RecoBTag.SecondaryVertex.pfSecondaryVertexTagInfos_cfi import * #pfSecondaryVertexTagInfos
+from RecoBTag.SecondaryVertex.pfInclusiveSecondaryVertexFinderTagInfos_cfi import * #pfInclusiveSecondaryVertexFinderTagInfos
+from RecoBTag.Combined.deepFlavour_cff import * #pfDeepFlavourTask
+
+makeDeepCSVForMiniAOD = pfDeepFlavourTask.copy()
+makeDeepCSVForMiniAOD.add(pfImpactParameterTagInfos)
+makeDeepCSVForMiniAOD.add(pfSecondaryVertexTagInfos)
+makeDeepCSVForMiniAOD.add(pfInclusiveSecondaryVertexFinderTagInfos)
+makeDeepCSVForMiniAOD.add(patJets)
+
+makePatJetsAndDeepCSVTask = makePatJets.copy()
```

Comment:
shouldn't this be ```makePatJetsAndDeepCSVTask = makePatJetsTask.``` ?
Also, since these additions are internal, please prefix with ```_``` so that they do not show up in modules including this file